minio/directpv
# Add drives command for listing, formatting, managing access-tiers, releasing drives

Steps to test 

- Set the ENVs or config file (refer https://github.com/minio/directpv/pull/651)
- Run through `kubectl directpv drives format` to add devices to DirectPV
- Verify if the formatted drives are created in `kubectl directpv drives list` output
- Manage the access tiers by `kubectl directpv drives access-tier` command
- Release the drives by `kubectl directpv drives release`
